node/blockAPI/baseBlock.go

```diff
@@ -126,3 +137,17 @@ func (bap *baseAPIBockProcessor) getFromStorerWithEpoch(unit dataRetriever.UnitT
 	storer := bap.store.GetStorer(unit)
 	return storer.GetFromEpoch(key, epoch)
 }
+
+func (bap *baseAPIBockProcessor) getBlockStatus(storerUnit dataRetriever.UnitType, blockAPI *blockStr.APIBlock) (string, error) {
```

Comment:
These structs are not tested at all. Please add a task to add unit tests.

Reply:
added this jira ticker for unit tests https://elrondnetwork.atlassian.net/browse/EN-8655